src/EditorFeatures/Core/Implementation/NavigateTo/AbstractNavigateToSearchService.SearchResult.cs

```diff
@@ -41,7 +41,7 @@ public SearchResult(Document document, DeclaredSymbolInfo declaredSymbolInfo, st
                 var declaredNavigableItem = navigableItem as NavigableItemFactory.DeclaredSymbolNavigableItem;
                 Debug.Assert(declaredNavigableItem != null);
 
-                _lazySummary = new Lazy<string>(() => declaredNavigableItem.Symbol.GetDocumentationComment()?.SummaryText);
+                _lazySummary = new Lazy<string>(() => declaredNavigableItem.Symbol?.GetDocumentationComment()?.SummaryText);
```

Comment:
Ideally, you're right, but since NavigateTo resolves the symbol as late as possible for perf reasons (i.e., my recent NavigateTo changes), we don't have the symbol (or null) until it's too late.  Realizing the symbol earlier in the process would knock our perf back down to what it was before.